### ISSUE TITLE
Generator crashes on non-asci characters

### ISSUE THREAD

**Issue by nielsdrost** [NLeSC/software.esciencecenter.nl#94]
If a non asci character is in an input file it crashes the generator :-(

**Comment by blootsvoets** [comment]
UTF-support in general seems lacking (#87)

**Comment by nielsdrost** [comment]
Stack trace:

```
Traceback (most recent call last):
  File "/usr/local/bin/estep", line 9, in <module>
    load_entry_point('estep==1.0.0', 'console_scripts', 'estep')()
  File "/home/niels/workspace/software.esciencecenter.nl/estep/script.py", line 222, in main
    generate_reciprocal()
  File "/home/niels/workspace/software.esciencecenter.nl/estep/script.py", line 175, in generate_reciprocal
    f.write(object2jekyll(document, 'description'))
  File "/home/niels/workspace/software.esciencecenter.nl/estep/format.py", line 49, in object2jekyll
    return "---\n{0}---\n{1}\n".format(metadata, content)
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2013' in position 395: ordinal not in range(128)
```

**Comment by blootsvoets** [comment]
Fixed by #111
